### ISSUE TITLE
Unable to startup rabbitmq using autocluster

### ISSUE THREAD

**Issue by JBodkin** [aweber/rabbitmq-autocluster#31]
Hello,

I'm experiencing difficulties attempting to get autocluster working correctly inside an docker container. The error message to me appears to be very confusing, would you be able to enlighten me?

```
rabbitmq:
  container_name: rabbitmq
  image: jbodkin/rabbitmq
  ports:
    - 4369:4369
    - 5672:5672
    - 15672:15672
    - 25672:25672
  links:
    - consul
```

Configuration
```
[
  {rabbit, [

    {loopback_users, []}

  ]},
  {autocluster, [
    {consul_host, "consul"},
    {consul_port, 8500}
  ]}
].
```

Log Entry 1
```
rabbitmq | Error description:
rabbitmq |    {could_not_start,rabbit,
rabbitmq |        {badarg,
rabbitmq |            [{erlang,list_to_integer,["tcp://172.17.0.3:8300"],[]},
rabbitmq |             {autocluster_util,as_integer,1,
rabbitmq |                 [{file,"src/autocluster_util.erl"},{line,34}]},
rabbitmq |             {autocluster_consul,nodelist,0,
rabbitmq |                 [{file,"src/autocluster_consul.erl"},{line,62}]},
rabbitmq |             {autocluster,ensure_registered,2,
rabbitmq |                 [{file,"src/autocluster.erl"},{line,97}]},
rabbitmq |             {autocluster,init,0,[{file,"src/autocluster.erl"},{line,27}]},
rabbitmq |             {rabbit,'-run_step/2-lc$^1/1-1-',1,[]},
rabbitmq |             {rabbit,run_step,2,[]},
rabbitmq |             {rabbit,'-run_boot_steps/1-lc$^0/1-0-',1,[]}]}}
```

Log Entry 2
```
{"init terminating in do_boot",{could_not_start,rabbit,{badarg,[{erlang,list_to_integer,["tcp://172.17.0.3:8300"],[]},{autocluster_util,as_integer,1,[{file,"src/autocluster_util.erl"},{line,34}]},{autocluster_consul,nodelist,0,[{file,"src/autocluster_consul.erl"},{line,62}]},{autocluster,ensure_registered,2,[{file,"src/autocluster.erl"},{line,97}]},{autocluster,init,0,[{file,"src/autocluster.erl"},{line,27}]},{rabbit,'-run_step/2-lc$^1/1-1-',1,[]},{rabbit,run_step,2,[]},{rabbit,'-run_boot_steps/1-lc$^0/1-0-',1,[]}]}}}
```

**Comment by gmr** [comment]
This looks similar to #18 in that in both cases, I'm seeing ``tcp://172.17.0.3:8300`` as what is being passed in as the port configuration value.

I'm wondering if there's an environment variable being set, in this case, ``CONSUL_PORT`` that has the ``tcp://172.17.0.3:8300`` value. 

Thanks for the docker-compose file, I'll try and reproduce with your setup.

**Comment by gmr** [comment]
Ok, figure this out. Docker container linking is breaking the ``CONSUL_PORT`` variable name:

```
CONSUL_NAME='/autocluster_rabbitmq_1/consul'
CONSUL_PORT='tcp://172.17.0.17:8300'
```

**Comment by gmr** [comment]
Addressed in 563acd3
